qt/python/mantidqt/widgets/test/test_fitpropertybrowser.py

```diff
@@ -110,37 +135,13 @@ def test_fit_result_workspaces_are_added_to_browser_when_fitting_done(self):
         self.assertEqual(name + "_Workspace", workspaceList.item(2).text())
 
     def test_fit_result_matrix_workspace_in_browser_is_viewed_when_clicked(self):
-        if not PYQT5:
-            self.skipTest("MatrixWorkspaceDisplay and TableWorkspaceDisplay cannot be "
-                          "imported in qt4 so the test fails with an error.")
-        from mantidqt.widgets.workspacedisplay.matrix.presenter import MatrixWorkspaceDisplay
-
-        name = "ws"
-        fig, canvas = self._create_and_plot_matrix_workspace(name)
-        property_browser = self._create_widget(canvas=canvas)
-        property_browser.setOutputName(name)
-
-        # create fake fit output results
-        matrixWorkspace = WorkspaceFactory.Instance().create("Workspace2D", NVectors=3, YLength=5, XLength=5)
-        AnalysisDataService.Instance().addOrReplace(name + "_Workspace", matrixWorkspace)
-
-        property_browser.fitting_done_slot(name + "_Workspace")
-        wsList = property_browser.getWorkspaceList()
-
-        # click on matrix workspace
-        MatrixWorkspaceDisplay.show_view = Mock()
-        item = wsList.item(0).text()
-        property_browser.workspaceClicked.emit(item)
-        self.assertEqual(1, MatrixWorkspaceDisplay.show_view.call_count)
-
-    def test_fit_parameter_table_workspaces_in_browser_is_viewed_when_clicked(self):
         if not PYQT5:
             self.skipTest("MatrixWorkspaceDisplay and TableWorkspaceDisplay cannot be "
                           "imported in qt4 so the test fails with an error.")
```

Comment:
This is dead code and can be removed, especially as it will incorrectly skip the test if we go to PyQt6